package.json

```diff
@@ -20,7 +22,25 @@
   },
   "homepage": "https://github.com/yralko/nodejs_gmp#readme",
   "dependencies": {
+    "@hapi/joi": "^16.1.8",
+    "@types/lodash.sortby": "^4.7.6",
+    "@types/uuid": "^3.4.6",
```

Comment:
@types/hapi__joi is missing, project won't compile without it

Reply:
I have this package in node_modules folder, however it's missing in the dependencies list. Thank you, added.